slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -403,6 +403,7 @@ void SLAMServiceImpl::CreateMap() {
                 }
             }
         }
+        VLOG(1) << "Passed sensor data to SLAM " << file;
```

Comment:
This log line is needed by the integration tests.

Reply:
[nit] I think it would be helpful to add a comment that clarifies that this log is needed for integration tests, such that someone doesn't accidentally delete it.

[q] Why do you use `VLOG(1)` here when we usually use `LOG(INFO)` in cartographer?